Save Inventory

danforbes opened the pull request:
Inventory can be archived by setting value of `saveInventoryFile` in webapp/services/properties.service.ts to `true`.

I don't think this is a solution to issue #12, but I think it's related.

Eric-Carlton commented:
@DanForbes I think if we could export via CSV instead of JSON, this would resolve #12. Something like [json2csv](https://www.npmjs.com/package/json2csv) may allow that out of the box, but I haven't done a ton of research.  

If you're interested, you could give that a shot. If not, I'll look into doing it at a later date. If you decide to just leave it as JSON, you can resolve the merge conflicts and I'll merge it in. I just messed with the PogoBuf version in package.json so I'm guessing that's what the merge conflict is. If so, the correct version should be `"^1.6.0"`

EDIT: I'll refrain from developing tonight so that we can prevent more merge conflicts before we decide what we want to accomplish with this PR.

Eric-Carlton commented:
@DanForbes actually [json-2-csv](https://www.npmjs.com/package/json-2-csv) may be easier to use. It doesn't look like it requires fields to be specified. It just takes an array of JSON objects (which we have) and spits out the CSV format.

dfoksma01 committed:
Save Inventory

Inventory can be archived by setting value of `saveInventoryFile` in webapp/services/properties.service.ts to `true`.

I don't think this is a solution to issue #12, but I think it's related.

Eric-Carlton commented:
Closing because I've implemented #12
